### DISCUSSION TITLE
What is the best practice for getting logged in User with custom model on server?

### DISCUSSION THREAD

**Discussion by igo** [nextauthjs/next-auth#1521] [Help]
**Your question**
How do I receive User instance with all custom attributes and related entities via TypeORM on the server side? I am using custom User model with extra attributes and relations. getSession returns very limited set of data

**What are you trying to do**
Get all User data stored in DB

**Reproduction**
`getSession({req})` returns only this:
```
{
  user: {
    name: string,
    email: string,
    image: uri
  },
  accessToken: string,
  expires: "YYYY-MM-DDTHH:mm:ss.SSSZ"
}
```

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by balazsorban44** [comment]
check the docs

https://next-auth.js.org/configuration/callbacks#jwt-callback

https://next-auth.js.org/configuration/callbacks#session-callback
